tests/kubeaddons-enterprise/kuttl-test.yaml

```diff
@@ -1,9 +1,23 @@
+---
 apiVersion: kudo.dev/v1alpha1
 kind: TestSuite
 commands:
-  - command: kubectl apply -f https://mesosphere.github.io/kubeaddons/bundle.yaml
-  - command: kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/v0.15.1/cert-manager.yaml
-testDirs: []
+  - command: docker cp kind-control-plane:/etc/kubernetes/pki/ca.crt /tmp/ca.crt
+  - command: docker cp kind-control-plane:/etc/kubernetes/pki/ca.key /tmp/ca.key
+  - command: >
+      bash -c "kubectl create namespace cert-manager --dry-run=client -o yaml |
+      kubectl apply -f -"
+  - command: >
+      bash -c "kubectl create secret tls kubernetes-root-ca
+      --namespace cert-manager --cert=/tmp/ca.crt --key=/tmp/ca.key
+      --dry-run=client -o yaml |
+      kubectl apply -f -"
+  - command: >
+      kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/v1.0.3/cert-manager.yaml
+  - command: >
+      kubectl wait --for=condition=ready pod -l app=webhook --timeout=5m --namespace cert-manager
```

Comment:
FYI the webhook ready does not mean the cert-manager is fully functional, that's why I wrote https://github.com/alenkacz/cert-manager-verifier :) (just saying).

Reply:
true, the bet here is on the time spent launching kubeaddons controller _after_ the webhook is up. if (or more like when) it blows up again we should consider a more solid approach.